docs/docs/models/td-full-metrics.md

```diff
@@ -0,0 +1,162 @@
+For these species, we will not be able to assess the performance of the model precisely.
+
+### Accuracy
+
+One of the ways we'll evaluate the model is classification accuracy, which is the percentage of videos
+that are classified correctly. Specifically, we computed:
+
+* Top-1 accuracy, which is the fraction of videos where the species the model gave the highest probability was correct.
+
+* Top-3 accuracy, which is the fraction of videos where one of the three species the model considered most likely was correct.
+
+Over all videos in the holdout set, the Top-1 accuracy is 82%; the Top-3 accuracy is 94%.
+As an example, if you chose a video at random from the holdout set, and the species with the highest probability is elephant,
+the probability is 82% that the video contains an elephant, according to the human-generated labels.
+If the three most likely species were elephant, hippopotamus, and cattle, the probability is 94% that the video
+contains at least one of those species.
+
+These results depend in part on the number of species represented in a particular dataset. For example, in the small
+number of videos from Equatorial Guinea, only three species appear. For these videos, the Top-1 accuracy is 97%, much
+higher than the overall accuracy.
+In the videos from Ivory Coast, 21 species are represented, so the problem is harder. For these
+videos, Top-1 accuracy is 80%, a little lower than the overall accuracy.
+
+### Recall and precision by species
+
+One of the goals of classification is to help with retrieval, that is, efficiently finding videos containing
+a particular species. To evaluate the performance of the model for retrieval, we can use
+
+* Recall, which is the fraction of videos containing a particular species that are correctly classified, and
+
+* Precision, which is the fraction of videos the model labels with a particular species that actually contain that species.
+
+The following figure shows recall and precision for the species in the holdout set.
+
+<img src="https://s3.amazonaws.com/drivendata-public-assets/zamba/td_full_set_precision_recall_by_species.png" alt="" style="width:800px;"/>
+
+It's clear that we are able to retrieve some species more efficiently than others. For example, elephants are relatively
+easy to find. Of the videos that contain elephants, 84% are correctly classified; and of the videos that the model
+labels "elephant", 94% contain elephants.
+So a researcher using the model to find elephant videos could find a large majority of them while viewing only a
+small number of non-elephant videos.
+
+Not surprisingly, smaller animals are harder to find. For example, the recall for rodent videos is only 22%.
+However, it is still possible to search for rodents by selecting videos that assign a relatively high probability
+to "rodent", even if it assigns a higher probability to another species or blank.
+
+
+### Removing blank videos
```

Comment:
We have also discussed what happens if you evaluate blank/non-blank while excluding the small species such as rodents, that we know researchers generally don't care about getting dropped. Curious if you took a look at that here. 

Also I love the interpretability of this graph from the user perspective. This is something we've been wanting to get at for a while

Reply:
We can definitely look into the results if we ignore certain small species. I'll make an issue to look into this as soon as we can, but I think we want to get this page posted today.